packages/admin/admin-icons/src/CheckboxIndeterminate.tsx

```diff
@@ -0,0 +1,15 @@
+import { SvgIcon, SvgIconProps } from "@mui/material";
+
+export default function CheckboxIndeterminate(props: SvgIconProps) {
```

Comment:
Why can't we use a SVG for this, like we do for the other icons?

Reply:
They aren't really Icons to be used as standard icons, the checkbox icons require custom svg-elements with classes, so we can style them correctly, depending on their state (like `disabled`) same with e.g., [CheckboxChecked](https://github.com/vivid-planet/comet/blob/main/packages/admin/admin-icons/src/CheckboxChecked.tsx). 

Arguably they shouldn't even be inside the `@comet/admin-icons` package, but directly in the `@comet/admin-theme` package where they are used, as there isn't really another use-case for these icons, as far as i am aware.